extensions/shared/premium-blocks/editor.scss

```diff
@@ -1,9 +1,70 @@
-// Premium Icons. Hide by default.
+@import 'modules/calypsoify/_studio-wpcom.scss';
+
+$banner-height: 48px;
+$icon-size: 10px;
+$icon-background-color: #fff;
+
+// Upgrade Banner.
+.jetpack-upgrade-plan-banner {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	font-size: 14px;
+	height: $banner-height;
+	line-height: 14px;
+	background: black;
+	color: white;
+	padding: 0 20px;
+	border-radius: 2px 2px 0 0;
+	margin-bottom: 10px;
+
+	.jetpack-upgrade-plan-banner__title,
+	.jetpack-upgrade-plan-banner__description {
+		margin-right: 10px;
+	}
+
+	.components-button {
+		flex-shrink: 0;
+		margin-left: auto;
+		height: 28px;
+
+		&.is-primary {
+			background: $studio-pink-40;
+
+			&:hover {
+				background: $studio-pink-30;
+			}
+		}
+	}
+}
+
+// Tweak Banner depending on the context.
+// Tweak Banner: Inspector Controls.
+.block-editor-block-list__block.is-upgradable {
+	margin-top: 0;
+	padding-top: $banner-height;
+}
+
+// Tweak Banner: Editor Canvas.
+.block-editor-block-list__layout .jetpack-upgrade-plan-banner {
```

Comment:
It seems (in my environment) that this rule is overridden, producing the following

![](https://cln.sh/uFxasz+)

Changing the rule to `.editor-styles-wrapper [data-block].is-upgradable {` fixes that for me. Can you try it out?

Reply:
let me take a look. What Gutenberg version are you using?

Reply:
Also, just in case, what theme are you using?

Reply:
![image](https://user-images.githubusercontent.com/77539/88797586-ef0bd700-d179-11ea-887f-f444cd76d946.png)

I caught it using other themes. 👍